holoviews/ipython/magics.py

```diff
@@ -231,21 +230,28 @@ class OutputMagic(OptionsMagic):
                                           'max-width', 'min-width', 'max-height',
                                           'min-height', 'outline', 'float']}}
 
-    defaults = OrderedDict([('backend'     , 'matplotlib'),
-                            ('fig'         , 'png'),
-                            ('holomap'     , 'widgets'),
-                            ('widgets'     , 'embed'),
-                            ('fps'         , 20),
+    defaults = OrderedDict([('backend'     , None),
+                            ('fig'         , None),
+                            ('holomap'     , None),
+                            ('widgets'     , None),
+                            ('fps'         , None),
                             ('max_frames'  , 500),
                             ('max_branches', 2),
-                            ('size'        , 100),
-                            ('dpi'         , 72),
+                            ('size'        , None),
+                            ('dpi'         , None),
                             ('charwidth'   , 80),
                             ('filename'    , None),
                             ('info'        , False),
-                            ('css'         , {})])
+                            ('css'         , None)])
+
+    # Defines the options the OutputMagic remembers the remainder
+    # is simply state on the backend specific Renderer
```

Comment:
'OutputMagic remembers the remainder is simply state'? Either punctuation or words are missing...